### PR TITLE
fix #70808

### LINKED ISSUE #70808: india.com
### Issue URL (Ads)
 [https://www.india.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.india.com%2F)
### Comment
Username: @singhalchirag1108
> that is a party ad i think it is a static ad that's why you have missed it 
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>Fanboy's Enhanced Tracking List<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media, <br/>Fanboy's Social Blocking List<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Chinese, <br/>EasyList Czech and Slovak

### PR DESCRIPTION
**Pull request by gork7777**
fix #70808 india.com

### CODE REVIEW

File: ChineseFilter/sections/specific.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ kocpc.com.tw##.kocpc_bot_ad
 ||store.7box.vip/ad/
 ||syndication.exosrv.com/splash.php?$xmlhttprequest,redirect=nooptext,important,domain=hboav.com
 sssam.com##.widget_html_ad
+india.com##.home-widget-leadStory
```
Comment (BlazDT) [inline ChineseFilter/sections/specific.txt:19]: Not an ad

File: ChineseFilter/sections/specific.txt [hunk 2]
```diff
@@ -74,4 +75,4 @@ nfmovies.com##a[href][onclick^="openurl()"] img[src^="/static/"]
 !+ PLATFORM(ios, ext_safari, ext_android_cb)
 nfmovies.com##body div[id^="aaaDiv"]
 !+ PLATFORM(windows, mac, android)
-||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
\ No newline at end of file
+||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
```
Comment (BlazDT) [inline ChineseFilter/sections/specific.txt:78]: Not seeing that picture in code.

**Comment by Yuki2718** [comment]
This is not a Chinese site in the first place.

File: ChineseFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply ChineseFilter/sections/specific.txt:19]: Yes, the issue can be closed unless OP gives more info.

Comment (zloyden) [reply ChineseFilter/sections/specific.txt:19]: > Not an ad

I agree with you.

File: ChineseFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply ChineseFilter/sections/specific.txt:78]: I also don't have this picture on my end.

**Pull request closed without merging**